radial_hit.m

```diff
@@ -1,5 +1,5 @@
-function[tMaxR, tStepR]=radial_hit(ray_origin, ray_direction, current_radial_voxel, ...
-        circle_center, circle_max_radius, delta_radius, r, t, verbose)
+function[tMaxR, tStepR, transitionFlag]=radial_hit(ray_origin, ray_direction, current_radial_voxel, ...
```

Comment:
Still needs documentation for input and output so the client understands what these represent.